zebra/zapi_msg.c

```diff
@@ -3144,12 +3144,29 @@ void zserv_handle_commands(struct zserv *client, struct stream_fifo *fifo)
 			goto continue_loop;
 		}
 
-		zapi_parse_header(msg, &hdr);
+		if (STREAM_READABLE(msg) < ZEBRA_HEADER_SIZE
```

Comment:
isn't this just `if (!zapi_parse_header)`?

Reply:
ah yeah